### DISCUSSION TITLE
Issue with Verbose Bootstrapping Output in PROCESS Macro When Generating PDF

### DISCUSSION THREAD

**Discussion by Xiaotian-Liu-MKT** [quarto-dev/quarto-cli#10131] [Q&A]
### Description

Hello everyone,

I’m encountering an issue with the output from the PROCESS Macro’s bootstrapping process when generating a PDF document in R Studio with Quarto. Here are the details:

Description of the Problem:

PROCESS Macro is a commonly used Macro to run mediating and moderating analysis in marketing and other social science areas.(https://www.processmacro.org/index.html)

I am using the PROCESS Macro to perform a mediating analysis, which involves a bootstrapping process. When I run this analysis, the entire bootstrapping process gets printed out in the PDF document, making it extremely verbose and hard to read.
<img width="1191" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/49262952/04b36487-54b3-4a85-83ff-144a731ea9e4">

However, when I run the same analysis in R normally, the results are presented in a much more concise and readable format.
<img width="620" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/49262952/47a94855-5b46-45f3-8318-c20510c76e46">

Is there a way to control the verbosity of the bootstrapping output when using the PROCESS Macro and generating a PDF?

The Code I use:
```{r}
process(data = df, y = "DV_Total", x = "Condition", m = "Judge", model = 4)
```

**Comment by mcanouil** [comment]
Unfortunately, we cannot help you efficiently with only screenshots.

> Is there a way to control the verbosity of the bootstrapping output when using the PROCESS Macro and generating a PDF?

That's unclear what you mean: either look at your tool documentation and/or https://quarto.org/docs/computations/execution-options.html
